addons/website_mass_mailing/views/snippets_templates.xml

```diff
@@ -60,6 +76,34 @@ database, without the s_newsletter_list class. See fixNewsletterListClass.
     </div>
 </template>
 
+<template id="s_newsletter_box" name="Newsletter Box">
+    <section class="s_newsletter_box s_newsletter_list o_colored_level pt64 pb64 o_cc o_cc2" data-name="Newsletter Box" data-list-id="0" data-oe-shape-data="{'shape':'web_editor/Blobs/04','colors':{'c5':'o-color-2'},'flip':[],'showOnMobile':false,'shapeAnimationSpeed':'0'}">
+    <div class="o_we_shape o_web_editor_Blobs_04" style="background-image: url('/web_editor/shape/web_editor/Blobs/04.svg?c5=o-color-2');"/>
+        <div class="container">
+            <div class="row s_nb_column_fixed s_col_no_bgcolor">
+                <div class="s_media_list_item col-lg-7 offset-lg-5 pt16 pb16" data-name="Box">
+                    <div class="s_col_no_resize s_col_no_bgcolor o_cc o_cc1 row g-0 align-items-center flex-row-reverse rounded overflow-hidden shadow-sm">
+                        <div class="s_media_list_img_wrapper col-lg-3 align-self-stretch">
+                            <img src="/web/image/website.library_image_07" class="s_media_list_img h-100 w-100 object-fit-cover" alt=""/>
+                        </div>
+                        <div class="s_media_list_body col-lg-9 p-3 p-md-5">
+                            <h4>Don't miss our upcoming updates</h4>
+                            <p class="text-muted">Get all the latest news, blog posts and product updates from our company, delivered directly to your inbox.</p>
+                            <p><br/></p>
+                            <t t-snippet-call="website_mass_mailing.s_newsletter_subscribe_form">
+                                <t t-set="_inputClasses" t-value="'o_subscribe_input'"/>
+                                <t t-set="thank_you_message">
+                                    <p class="h6-fs text-success"><i class="fa fa-check-circle-o" role="img"/> Thanks for registering!</p>
```

Comment:
I don't find the icon really necessary... up to you tho ;) 

```suggestion
                                    <p class="h6-fs text-success">Thanks for registering!</p>
```
ps: Sorry, I forgot to write a note about it during by prev. review.